ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -211,20 +217,44 @@ public ConsistencyCheckerTool(ClusterMap clusterMap, StoreKeyFactory storeKeyFac
     return new Pair<>(sane, results);
   }
 
+  /**
+   * Takes all the StoreKeys from the file replicas and runs them
+   * through the StoreKeyConverter in a batch operation
+   * @param replicas An Array of replica directories from which blobIds need to be collected
+   * @param results the results of processing the indexes of the given {@code replicas}.
+   * @return mapping of original keys to converted keys.  If there's no converted equivalent
+   * the value will be the same as the key.
+   * @throws Exception
+   */
+  private Map<StoreKey, StoreKey> createConversionKeyMap(File[] replicas,
+    Map<File, DumpIndexTool.IndexProcessingResults> results) throws Exception {
+    List<StoreKey> storeKeys = new ArrayList<>();
```

Comment:
Isn't a `Set` better here? The keys will be repeated across stores.

Reply:
fixed